.github/workflows/build.yml

```diff
@@ -162,10 +162,35 @@ jobs:
         python -m pip install -U setuptools wheel pip
         python -m pip install "cmake==${WABT_CMAKE_VER}.*"
         cmake --version
-    - name: Configure WABT
+
+    - name: configure (OpenSSL)
       run: cmake -G Ninja -S . -B out -DCMAKE_BUILD_TYPE=Release
-    - name: build
+    - name: build (OpenSSL)
       run: cmake --build out
+    - name: install (OpenSSL)
+      run: cmake --install out --prefix _opt/wabt-openssl
```

Comment:
What is `_opt` here?  How about `install/...`?

Reply:
Or how about `install` and install-picocha` to match the `out` names?

Reply:
Done!